src/UglyToad.PdfPig/Content/PageTreeMembers.cs

```diff
@@ -6,17 +6,26 @@
     /// <summary>
     /// Contains the values inherited from the Page Tree for this page.
     /// </summary>
-    internal class PageTreeMembers
+    public sealed class PageTreeMembers
     {
-        public CropBox GetCropBox()
+        internal CropBox GetCropBox()
         {
             return null;
         }
 
+        /// <summary>
+        /// The page media box.
+        /// </summary>
         public MediaBox MediaBox { get; set; }
```

Comment:
Could these property setters be internal to stop consumers messing with them?

Reply:
well spotted, now updated